core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/api/TrackProperties.kt

```diff
@@ -59,11 +60,17 @@ interface TrackProperties {
         trackChunk: TrackChunkId
     ): StaticIdxList<OperationalPointPart>
 
-    fun getOperationalPointPartChunk(operationalPoint: OperationalPointPartId): TrackChunkId
+    fun getOperationalPointPartChunk(operationalPointPart: OperationalPointPartId): TrackChunkId
 
     fun getOperationalPointPartChunkOffset(
-        operationalPoint: OperationalPointPartId
+        operationalPointPart: OperationalPointPartId
     ): Offset<TrackChunk>
 
-    fun getOperationalPointPartName(operationalPoint: OperationalPointPartId): String
+    fun getOperationalPointPartOpId(operationalPointPart: OperationalPointPartId): String
+
+    fun getOperationalPointPartProps(
+        operationalPointPart: OperationalPointPartId
+    ): OperationalPointPartProps
+
+    fun getOperationalPointPartKp(operationalPointPart: OperationalPointPartId): String?
```

Comment:
I'm aware that the current data model of RawInfra does not quite match the data model of RailJSON: in the new infra API, there are just OperationalPointPart, no OperationalPoint.

That's because merging both types is, as far as I know, the eventual target for RailJSON: parent operational points are only useful as repositories for shared metadata. the issue is: shared using what criteria? we would rather have a single type, and have one operational point that's actually a point than go on with the current mess.

But right now, there's a mismatch between the two APIs. Here are some ways it could be solved:
- the eventual target (removing the OperationalPointPart object and having one OperationalPoint per track location) is too costly to do now
- adopt the RailJSON model and migrate later
- perform the de-normalization step (adding the metadata of operational points into parts) on import

I believe the last option has the best long-term viability, and is also the easiest to implement. However, I don't really like how it's currently done:
- despite having performed de-normalization, there is a separate call for the kilometric point (getOperationalPointPartKp)
- RailJSON extensions leak into the SimInfra API, even though they could really be opaque (data is just passed around, not read)

I would instead suggest the following:

```suggestion
    // returns a unique string identifier for this object
    fun getOperationalPointPartName(operationalPointPart: OperationalPointPartId): String

    fun getOperationalPointPartProps(
        operationalPointPart: OperationalPointPartId
    ): Map<String, String>
```

during RailJSON parsing, the name of the operationalpointpart would be built as follows: `${operationalPoint.name}.${operationalPointPart.index}`

Props would be a generic metadata storage facility, which could be later used for querying objects having some property value (indices can be built lazily). When returning operational points on the path, properties can be parsed back for editoast consumption.

Reply:
I separated kp from the rest because it's an operationalPoint prop, whereas the others were operationalPointPartProps. This would be a bit more complex, but would globally work out, so why not.